qemu/tests/cfg/block_resize.cfg

```diff
@@ -10,9 +10,7 @@
     create_image_stg = yes
     force_create_image_stg = yes
     blk_extra_params_stg = "serial=TARGET_DISK0"
```

Comment:
In fact, I want to remove blk_extra_params_stg.
But I'm not sue if it will affect you-Redhat(er).
Hope feedback here！！！